_posts/2022/2022-02-17-lets-talk-about-numbers-2022.md

```diff
@@ -0,0 +1,20 @@
+---
+layout: post
+title: Let's talk about numbers 2022
+date: 2022-02-17
+header_image: private/survey-header.png
+header_overlay: true
+category: on-the-job
+tags: ["company culture", "facts", "survey"]
+authors: ["Alicia", "Anne"]
+about_authors: ["aschroeder", "azimmermann"]
+---
+
+Some time has passed since we posted our last [Let's talk about numbers](/blog/on-the-job/lets-talk-about-numbers-2020/){:target="_blank"} in 2020.
```

Comment:
```suggestion
Some time has passed since we posted our last [Let's talk about numbers](/blog/on-the-job/lets-talk-about-numbers-2020/){:target="_blank"} blog post in 2020.
```